src/core/scenario.jl

```diff
@@ -111,22 +111,77 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
+                            )
     end
 
     if haskey(d, "ElectricUtility") && !(settings.off_grid_flag)
-        electric_utility = ElectricUtility(; dictkeys_tosymbols(d["ElectricUtility"])...)
+        electric_utility = ElectricUtility(; dictkeys_tosymbols(d["ElectricUtility"])...,
+                                            latitude=site.latitude, longitude=site.longitude, 
+                                            time_steps_per_hour=settings.time_steps_per_hour
+                                        )
     elseif !(settings.off_grid_flag)
-        electric_utility = ElectricUtility()
+        electric_utility = ElectricUtility(; latitude=site.latitude, longitude=site.longitude, 
+                                            time_steps_per_hour=settings.time_steps_per_hour
+                                        )
     elseif settings.off_grid_flag 
         if haskey(d, "ElectricUtility")
             @warn "ElectricUtility inputs are not applicable when off_grid_flag is true and any ElectricUtility inputs will be ignored. For off-grid scenarios, a year-long outage will always be modeled."
         end
-        electric_utility = ElectricUtility(; outage_start_time_step = 1, outage_end_time_step = settings.time_steps_per_hour * 8760) 
+        electric_utility = ElectricUtility(; outage_start_time_step = 1, 
+                                            outage_end_time_step = settings.time_steps_per_hour * 8760, 
+                                            latitude=site.latitude, longitude=site.longitude, 
+                                            time_steps_per_hour=settings.time_steps_per_hour
+                                        ) 
     end
 
+    #Handle missing emissions inputs (due to failed lookup and not provided by user)
+    #TODO: condition below code on: if site.off_grid == false
+    if isnothing(electric_utility.emissions_factor_series_lb_CO2_per_kwh)
+        if (!isnothing(site.CO2_emissions_reduction_min_pct) || 
+            !isnothing(site.CO2_emissions_reduction_max_pct) || 
+            settings.include_climate_in_objective)
+            throw(@error "To include CO2 costs in the objective function or enforce emissions reduction constraints, you must either enter custom health emissions factors or a site location within the continental U.S.")
+        else
+            electric_utility.emissions_factor_series_lb_CO2_per_kwh = zeros(8760*settings.time_steps_per_hour)
+        end
+    end
+    for emissions_type in ["NOx", "SO2", "PM25"]
+        if isnothing(getproperty(electric_utility, Symbol("emissions_factor_series_lb_$(emissions_type)_per_kwh")))
+            if settings.include_health_in_objective
+                throw(@error "To include health costs in the objective function, you must either enter custom health emissions factors or a site location within the continental U.S.")
+            else
+                setproperty!(electric_utility, 
```

Comment:
These defaults of zero emissions factors should be done in the `ElectricUtility` constructor. Then we can keep the `ElectricUtility` immutable and we won't need the `Union` types.

Reply:
This has been addressed in `emissions_series` where default returned is a vector of `zeros(8760*time_steps_per_hour)`

Reply:
I think we need to go back to returning nothing from emissions_series if it fails, because if that happens and the user requested an emissions reduction or emissions cost in the objective then we need to error. If the user didn't do either of those, then we can emissions factors to zeros. I also need to bring back the code I wrote to do this check that you commented out, but I will put it in the ElectricUtility struct instead of Scenario so that ElectricUtility can be immutable.

Reply:
done in commit 5a22af9